pages/contact-legislators.js

```diff
@@ -35,38 +33,33 @@ function ContactLegislators() {
         setOfficials(officials)
     }
     return (
-        <div>
-            <Head>
-                <title>Contact Your Legislators - { site.name }</title>
-            </Head>
-            <SharedLayout>
-                <BackButton className="mt-3 mb-2" />
-                <h1 className="subpage-header">Contact Your Legislators</h1>
-                <Formik
-                    initialValues={{}}
-                    onSubmit={submitZipCode}
-                >
-                    <Form className="col-9">
-                        <Input
-                            label="Enter your zip code or full address:"
-                            name="zip_code"
-                            required={ true }
-                            smallText='Your information will not be stored or used in any way except to show you your elected officials. For state elected officials in your district, please use your full street address.'
-                            className="mb-0"
-                        />
-                        <Submit />
-                    </Form>
-                </Formik>
-                <div className='mb-3 pl-3' >
-                    {officials.map(o => <Official key={o.name} official={o} />)}
-                </div>
-                <div className='mb-5 pl-3' >
-                    <strong>Sample phone message</strong>
-                    <p>Hello, (optional: my name is __, and) I am a constituent in your district. The #1 obstacle to safety for survivors of intimate partner violence is financial insecurity. I am calling because [state] can and must do more to help survivors build the financial security they need to stay safe. I urge you to visit FreeFrom’s National Financial Security Policy Map and Scorecard to see specific recommendations for how we can make policy changes to prioritize financial security and long-term safety for survivors in our state. Visit FreeFrom.org and check out the map to learn more. Thank you.</p>
-                </div>
-                <TakeAction />
-            </SharedLayout>
-        </div>
+        <SharedLayout title="Contact Your Legislators">

```

Comment:
This looks like a huge diff because I removed a `div` that is no longer necessary and un-indented this whole component. The only change is the title.